client/app/pages/destinations/DestinationsList.jsx

```diff
@@ -60,12 +65,7 @@ class DestinationsList extends React.Component {
         Destination.query().then(destinations => this.setState({ destinations, loading: false }));
         return destination;
       })
-      .catch(error => {
-        if (!(error instanceof Error)) {
-          error = new Error(get(error, "data.message", "Failed saving."));
```

Comment:
In this specific case this was intentional :grimacing:, the error message "Alert Destination with the name X already exists." comes from it

Reply:
Omg... Okay, I'll restore it, thank you!